Sources/ForageSDK/Foundation/Network/Model/ForageXKeyModel.swift

```diff
@@ -10,4 +10,5 @@ import Foundation
 /// `ForageXKeyModel` used for compose X-key request
 internal struct ForageXKeyModel: Codable {
     let alias: String
+    let bt_alias: String
```

Comment:
So the encryption_alias endpoint now returns both `alias` and `bt_alias`? Just confirming for the other repos.